src/main/java/org/openstreetmap/atlas/generator/AtlasGenerator.java

```diff
@@ -398,4 +421,17 @@ private void saveAsHadoop(final JavaPairRDD<?, ?> atlasRDD, final AtlasGenerator
                 new JobConf(configuration()));
         logger.info(SAVED_MESSAGE, group.getDescription());
     }
+
+    private Sharding sharding(final String shardingName, final CommandMap command)
+    {
+        if (shardingName == null)
+        {
+            return persistenceTools()
+                    .sharding((String) command.get(AtlasGeneratorParameters.PBF_PATH));
```

Comment:
Why do we default to the Pbf path here? It won't contain the sharding tree name, and the AtlasSharding.forString() on like 434 will always fail, no?

Reply:
So this method here is for the Atlas sharding. If the parameter is not provided, it will use the PersistenceTools to load it from the input PBF folder. Does that make sense?